docs/develop/json-manifest-overview.md

```diff
@@ -0,0 +1,870 @@
+                                            "description": "click to open dashboard"
+                                        },
+                                        "actionId": "text.showDashboard"
+                                    }
+                                ]
+                            }
+                        ]
+                    }
+                ]
+            },
+            {
+                "contexts": [
+                    "readMail"
+                ],
+                "tabs": [
+                    {
+                        "nativeId": "tabDefault",
+                        "groups": [
+                            {
+                                "id": "dashboard",
+                                "label": "Controls",
+                                "controls": [
+                                    {
+                                        "id": "showDashboard",
+                                        "type": "button",
+                                        "label": "Show dashboard",
+                                        "icons": [
+                                            { "size": 16, "file": "test_16.png" },
+                                            { "size": 32, "file": "test_32.png" },
+                                            { "size": 80, "file": "test_80.png" }
+                                        ],
+                                        "supertip": {
+                                            "title": "Show dashboard",
+                                            "description": "click to open dashboard"
+                                        },
+                                        "actionId": "text.showDashboard"
+                                    }
+                                ]
+                            }
+                        ]
+                    }
+                ]
+            }
+        ],
+        "keyboards": [
+            {
+                "requirements": {
+                    "capabilities": [
+                        { "name": "SharedRuntime", "minVersion": "1.1" }
+                    ],
+                    "platforms": ["windows", "web"]
+                },
+                "shortcuts": [
+                    {
+                        "key":  "Ctrl+Shift+U",
+                        "actionId": "text.toUppercase"
+                    },
+                    {
+                        "key": "Ctrl+Shift+L",
+                        "actionId": "text.toLowercase"
+                    },
+                    {
+                        "key": "Ctrl+Shift+Up",
+                        "actionId": "text.showDashboard"
+                    }
+                ]
+            },
+            {
+                "requirements": {
+                    "capabilities": [
+                        { "name": "SharedRuntime", "minVersion": "1.1" }
+                    ],
+                    "platforms": ["mac"]
```

Comment:
Right now it doesn't work easily on mac, so I think we can take this part out.
The difference for mac is that installing `.net` dependency is much harder and we won't help customers with that in the tooling